### DISCUSSION TITLE
just a simple question

### DISCUSSION THREAD

**Discussion by eafv** [Prodigy-Hacking/ProdigyMathGameHacking#1315] [Questions] [answered]
we all had this talk about this 418 error but if its caused by buddies in general, why not remove it by a command or something

**Comment by afkvido** [comment, marked as answer]
418 is unavoidable. If you'd like to see a full list of actions that trigger 418, see [this](https://github.com/ProdigyErrorCodes/ProdigyErrorCodes/blob/errors/!%20LIST.md)
